pkg/vm/json_test.go

```diff
@@ -108,6 +108,23 @@ func TestUT(t *testing.T) {
 	require.Equal(t, true, testsRan, "neo-vm tests should be available (check submodules)")
 }
 
+func getTestingInterop(id uint32) *InteropFuncPrice {
+	// FIXME in NEO 3.0 it is []byte{0x77, 0x77, 0x77, 0x77} https://github.com/nspcc-dev/neo-go/issues/477
+	if id == InteropNameToID([]byte("Test.ExecutionEngine.GetScriptContainer")) {
```

Comment:
from the first glance it's very hard to recognize difference between these 2 parts. 
I think it's niece to have at least godoc.

Reply:
We probably can merge them, will do.